framework/src/task_manager/task_manager_core.c

```diff
@@ -977,7 +977,25 @@ static int taskmgr_is_tm_broadcast_msg_init(void)
 	return 1;
 }
 
-static int taskmgr_set_msg_cb(int type, void *data, int pid)
+static int taskmgr_set_unicast_cb(tm_unicast_callback_t data, int pid)
+{
+	int handle;
+
+	if (data == NULL) {
```

Comment:
how about checking data is NULL or not before calling taskmgr_set_unicast_cb?

Reply:
I think checking the input of function is better even though there is no possibility of data lost.